topi/python/topi/x86/conv2d_avx_common.py

```diff
@@ -36,7 +38,12 @@ def _get_default_schedule(wkl, simd_width):
             reg_n = n
             break
 
-    return AVXConvCommonFwd(ic_bn, oc_bn, reg_n, False)
+    parallel_chunk = min(multiprocessing.cpu_count()//2, 1)
```

Comment:
Why are you using min here instead of max?

Reply:
typo... sorry.